cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -490,7 +540,12 @@ def act_sigmoid_scaled(x):
     else:
         K.set_session(tf.Session(config=tf.ConfigProto(device_count={"GPU": 0})))
 
-    # build the model
+    # build the Keras model with:
+    # 1. a layer to concatenate the list of inputs
```

Comment:
We have two layers, correct? Also, we have an embedding layer before the concatenate layer. 

We need not really explain the model arch. It's okay to skip this.